regression/esbmc-unix/03_boundedBuffer/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+KNOWNBUG
```

Comment:
The label claims this bug is known. It is not, though, at least to me. :)
@brcfarias Could you please open an issue so we know what it's about and where it came from?